Workout/SwimmingWorkout.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  SwimmingWorkout.swift
+//  Workout
+//
+//  Created by Marco Boschi on 04/03/2017.
+//  Copyright © 2017 Marco Boschi. All rights reserved.
+//
+
+import Foundation
+import HealthKit
+import MBLibrary
+
+class SwimmingWorkout: Workout {
+	
+	override init(_ raw: HKWorkout, delegate del: WorkoutDelegate?) {
+		super.init(raw, delegate: del)
+		
+		if #available(iOS 10, *) {
+			self.addDetails()
+			self.addRequest(for: HKQuantityTypeIdentifier.distanceSwimming.getType()!, withUnit: .kilometer(), andTimeType: .ranged, searchingBy: .workout)
```

Comment:
in general, swimming workouts are expressed in meters.
As an example, for one session, distance may vary between a few 100s of meters for beginners to a something like 4500 meters for semi-pros.